.storybook/recipes/CalendarCard/CalendarCard.tsx

```diff
@@ -0,0 +1,119 @@
+import clsx from 'clsx';
+import React, { ReactNode } from 'react';
+import styles from './CalendarCard.module.css';
+
+import { Card, Heading, Icon, Tag, Text } from '../../../src';
+import { Variant as TagVariant } from '../../../src/components/Tag/Tag';
+
+export const VARIANTS = ['brand', 'revise', 'success'] as const;
+
+export type Variant = typeof VARIANTS[number];
+
+export interface Props {
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+  /**
+   * Card sub components that are nested within the Card wrapper.
+   */
+  children?: ReactNode;
+  /**
+   * End of date range
+   */
+  dateEnd?: string;
+  /**
+   * Start of date range
+   */
+  dateStart?: string;
+  /**
+   * Additional text for date range
+   */
+  meta?: string;
+  /**
+   * Text for tag(s)
+   */
+  tags?: TagType[];
+  /**
+   * The title text
+   */
+  title: string;
+  /**
+   * Notification variant for the card.
+   */
+  variant?: Variant;
+}
+
+interface TagType {
+  text: string;
+  variant?: TagVariant;
+  hasOutline?: boolean;
+}
+
+/**
+ * Recipe for a Card component to represent an event within calendar UI
+ */
+export const CalendarCard = ({
+  children,
+  className,
+  dateEnd,
+  dateStart,
+  meta,
+  tags,
+  title,
+  variant = 'brand',
+  ...other
+}: Props) => {
+  const componentClassName = clsx(
+    className,
+    styles['calendar-card'],
+    styles[`calendar-card--${variant}`],
+  );
+  return (
+    <Card className={componentClassName} {...other}>
+      <Heading
```

Comment:
We always want to use at least `CardBody` as a child component. This allows the card to take up the entire space if put inside a grid. So let's wrap all of the content in `CardBody`. `CardHeader` and `CardFooter` are always optional, but `CardBody` should be required.